### ISSUE TITLE
game name and seo problems

### ISSUE THREAD

**Issue by yoel123** [yoel123/java-text-based-space-station-manger-sim#53]
as of now, there is no short search term (or any google search term) that will lead to this repository. meaning no organic traffic at all.

one of the reasons is that the game has no name, if it had one it could be found by typing the game name.

the search term "space station tycon" or "java" or "text based game" are very competitive, "open source" is also very competitive.

not having a name means you cant send people to search for the game by name, what is I good for?

for example in quora, if you post too much of the same link hey will ban you and your link (meaning no one will be able to post your link), the same is with Facebook and also Twitter (also facebook reduces exposure if you add a link to your post).

but if you tell people "search <game name> on google" you will most likely go under the radar as those systems hunt for spam links.

but the game has no name.

I'm not good at naming things, I rather pay someone to come up with a name then have someone come 5 years later saying "you owe me money I came up with the name!" personally I hate people like that, who want money for other peoples hard work just because they "came up with an idea".

in short, if you have a name suggestion, I will add you to the credits if I use it, but don't expect to get anything for it.
